### DISCUSSION TITLE
Icl support

### DISCUSSION THREAD

**Discussion by CoolHandDevil505** [BazzaCuda/CustomMenu#15] [Ideas]
Hi!

Me again 😊

Any plans for supporting .icl libraries?

Thanks a lot.

**Comment by CoolHandDevil505** [comment]
It is supported, like any other dll, works fine.
It is not specified that icl are supported on the config menu, maybe add the mention?

**Reply by BazzaCuda** [reply]
Icon libraries. That's a blast from the past. Well, for me, anyway. I didn't realise they were still a thing.
It's good to know they're already supported by CustomMenu without any additional work from me :P 
I'll see what I can do to make that clear in the config window.

**Reply by BazzaCuda** [reply]
I've made it more obvious in the config window that ICLs are supported, and in the Icon Explorer window I've added a filter specifically for ICL files.

**Reply by CoolHandDevil505** [reply]
Nice! 👌
In case you don't know, this software is really convenient to browse through dlls, exe and libraries : [Icon Explorer](http://mitec.cz/iconex.html)

**Reply by BazzaCuda** [reply]
I use [IcoFX](https://icofx.ro) myself.
But I decided it would be really inconvenient to make CustomMenu users use other software to first extract their desired icon(s) into .ICO files and _then_ have to configure CustomMenu to use those ICO files. Hence why I wrote CustomMenu's own Icon Explorer window.
